src/User/resources/views/settings/account.php

```diff
@@ -102,6 +102,21 @@
                     <p>
                         <?= Yii::t('usuario', 'Two factor authentication protects you in case of stolen credentials') ?>.
                     </p>
+                    <?php if ($module->enableTwoFactorAuthentication && !$model->getUser()->auth_tf_enabled):  
+                        $validators = $module->twoFactorAuthenticationValidators; 
+                        foreach( $validators as $name => $validator ) {
+                            $description = $validator[ "description" ];
+                            $checked = $name=='google-authenticator'?'checked':'';
```

Comment:
Why is this check done only against *google-authenticator*?

Reply:
This is the default value checked in case the user doesn't yet enabled 2FA.